src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c

```diff
@@ -71,26 +71,26 @@ bool ReadAzureDPS2MqttSettings(AZURE_DPS_2_MQTT_SETTINGS* settings)
     if (!ADUC_AgentInfo_ConnectionData_GetStringField(agent_info, "dps.registrationId", &settings->registrationId))
     {
         // TODO (nox-msft) : read common name from certificate.
-        Log_Error("Cannot read registrationId field.");
+        Log_Error("get registrationId");
         goto done;
     }
 
     // For DPS connection, the clientId and registrationId fields are the same.
     if (mallocAndStrcpy_s(&settings->mqttSettings.clientId, settings->registrationId) != 0)
     {
-        Log_Error("Cannot allocate clientId field.");
+        Log_Error("alloc clientId");
```

Comment:
not much value logging on OOM.  If we're really OOM, logging would probably fail too.